hathor/manager.py

```diff
@@ -98,6 +99,7 @@ def __init__(self,
                  event_manager: EventManager,
                  feature_service: FeatureService,
                  bit_signaling_service: BitSignalingService,
+                 feature_storage: FeatureActivationStorage | None,
```

Comment:
Why optional? Shouldn't it be part of the `BitSignalingService`?

Reply:
It's optional because when the full node is running with `--memory-storage`, this check doesn't make sense. As this is only here to prevent human error in the configuration file, it's only necessary when running the full node "for real", with a rocksdb storage.

Also, I left it separate from the `BitSignalingService` for the same reason, as this is just a pre-initialization check. Maybe the name `FeatureActivationStorage` is not so good, it could be something more specific like `FeatureActivationSettingsValidator`?

Reply:
Got it. But why would HathorManager have to understand the validation of the feature service? Shouldn't the FeatureService or the BitSignalingService do it themselves? Maybe they will have to have a `start()` method?

Reply:
I moved it to the `BitSignalingService` on https://github.com/HathorNetwork/hathor-core/pull/894/commits/dfb7c66a17ae0452453ea1ffe4bbb9307a19676e

Reply:
Thanks!